x-pack/test/security_solution_endpoint/apps/endpoint/responder.ts

```diff
@@ -57,13 +102,79 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
 
       it('should show Responder from the endpoint list', async () => {
-        await pageObjects.endpoint.showResponderFromEndpointList(indexedData.hosts[0].agent.id);
+        await pageObjects.endpoint.showResponderFromEndpointList(endpointAgentId);
         await performResponderSanityChecks();
       });
 
       it('should show Responder from the endpoint details', async () => {
-        await pageObjects.endpoint.showResponderFromEndpointDetails(indexedData.hosts[0].agent.id);
+        await pageObjects.endpoint.showResponderFromEndpointDetails(endpointAgentId);
+        await performResponderSanityChecks();
+      });
+    });
+
+    describe('from timeline', () => {
+      let timeline: TimelineResponse;
+
+      before(async () => {
+        timeline = await timelineTestService.createTimeline('endpoint responder test');
+
+        // Add all alerts for the Endpoint to the timeline created
+        timeline = await timelineTestService.updateTimeline(
+          timeline.data.persistTimeline.timeline.savedObjectId,
+          {
+            title: timeline.data.persistTimeline.timeline.title,
+            kqlQuery: {
+              filterQuery: {
+                kuery: {
+                  kind: 'kuery',
+                  expression: `agent.type: "endpoint" AND agent.id : "${endpointAgentId}" AND kibana.alert.rule.uuid : *`,
+                },
+                serializedQuery: getEndpointAlertsQueryForAgentId(endpointAgentId).$stringify(),
+              },
+            },
+          },
+          timeline.data.persistTimeline.timeline.version
+        );
+
+        await detectionsTestService.waitForAlerts(
+          getEndpointAlertsQueryForAgentId(endpointAgentId),
+          // The Alerts rules seems to run every 5 minutes, so we wait here a max
```

Comment:
If anyone from the Threat Hunting team know of a way I can explicitly trigger the rule to run via API let me know. I would avoid us having to wait at least 5 minutes for it to run and create the alerts